io-actions/src/fragment/translate.js

```diff
@@ -0,0 +1,44 @@
+const fetch = require('node-fetch');
+
+const PATH_TOKENS =
+    /\/content\/dam\/mas\/(?<surface>[\w]+)\/(?<parsedLocale>[\w_]+)\/(?<fragmentPath>.*)/;
+
+/**
+ * we expect a body to already have been fetched, and a locale to be requested
+ */
+async function main({ status, body, locale }) {
+    if (!status || status != 200 || !body || !locale) {
+        return {
+            status: 400,
+        };
+    }
+    const match = body.path?.match(PATH_TOKENS);
+    if (!match) {
+        return {
+            status: 400,
```

Comment:
lets add the error reason? smth like
`reason: 'Fragment Path ${body.path} doesn't match the regex',`